fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -301,6 +302,15 @@ open class DefaultToolbarMenu(
         onItemTapped.invoke(ToolbarMenu.Item.SaveToCollection)
     }
 
+    private val printPageItem = BrowserMenuImageText(
+        // ToDo: Separate strings or not for menu and share?
```

Comment:
Should we file a follow up ticket for this? :)

Reply:
I planned on resolving it in this patch! 😁 I wasn't sure what the standard practice is for strings in Fenix. 

I doubt the naming will ever be different for "Print" from Share versus "Print" from Toolbar, but I could also see it being a good practice to always have `one string = one UI element`. 

Do you have a recommendation of having two separate "Print" strings or just one?

Reply:
I think for the moment it's ok, as both are the same, if we need to change we could update it the feature.

Reply:
Thanks, removed comment and left it with one string for both menus!